otlp/go.mod

```diff
@@ -3,15 +3,15 @@ module go.opentelemetry.io/proto/otlp
 go 1.17
 
 require (
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.23.0
-	google.golang.org/grpc v1.67.1
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.24.0
+	google.golang.org/grpc v1.68.0
```

Comment:
I think that there is nothing in the build that tests if the generated code actually builds

Shouldn't `go 1.17` be updated?

See: https://github.com/grpc/grpc-go/blob/v1.68.0/go.mod#L3

I think that renovate (instead of dependabot) would be able to bump the Go version.